indico/types/filetype.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+
+class FileType(Enum):
```

Comment:
don't get me wrong, I love an enum, but the SDK does become a little more brittle. for example, if we introduce zip support in the platform, a user will need to upgrade their SDK version whereas before, they'd be able to create a zip sub and retrieve it in queries without any problems

although... to be fair, I'm not sure I see where you use this enum?

Reply:
whoops yeah I ended up using `str` instead for that reason but forget to remove the unused enum definition. Removing now